xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,487 @@
+                        vote is provided to the largest partition. This
+                        algorithm is useful when the cluster with only one node
+                        should remain quorate. </para>
+                  </listitem>
+                  <listitem>
+                     <para>
+                        For FFSplit (<quote>fifty-fifty split</quote>), this
+                        algorithm provides one node and behaves like another
+                        node. It is used for even-node clusters. The quorum
+                        server can disappear. </para>
+                  </listitem>
+               </itemizedlist>
+            </listitem>
+         </varlistentry>
+         <varlistentry>
+            <term>Heuristics</term>
+            <listitem>
+               <remark>toms 2020-32-11: this is taken from the corosync-qdevice
+               manpage. Not sure how to express that better.
+               </remark>
+               <para>
+                  &qdevice; supports a set of commands executed locally on startup,
+                  cluster membership change, successful connect to <systemitem
+                     class="daemon">corosync-qnetd</systemitem>, or optionally at
+                  regular times.
+               </para>
+               <para>
+                  When all commands were executed successfully, the heuristics have
+                  passed, otherwise they have failed. The heuristic result is
+                  sent to <systemitem class="daemon">corosync-qnetd</systemitem>
+                  and there it's used in calculations to determine which partition
+                  should be quorate.
+               </para>
+            </listitem>
+         </varlistentry>
+         <varlistentry>
+            <term>Tiebreaker</term>
+            <listitem>
+               <remark>toms 2020-05-11: do we need to explain it here?</remark>
+               <para></para>
+            </listitem>
+         </varlistentry>
+      </variablelist>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-require">
+      <title>Requirements and Prerequisites</title>
+      <para>
+         Before you start using &qdevice; and &qnet;, make sure the following
+         conditions are fullfilled:
+      </para>
+      <itemizedlist>
+         <listitem>
+            <para>
+               An already functional cluster, consisting of different cluster nodes.
+            </para>
+         </listitem>
+         <listitem>
+            <para>
+               An additional machine which will become the &qnet; server.
+               You only need to install &sls; &productnumber;, but not the &hasi;.
+            </para>
+         </listitem>
+         <listitem>
+            <para>
+               A network address which is not part of the network
+               address range of your cluster. This is recommended for the
+               &qnet; server.
```

Comment:
I can't recall where I did read it. Do you think I should remove that as it may confuse our customers? Or leave it as a (mild) recommendation?

Reply:
It's just difficult to cover different situations ... But we could simply say "The location of corosync-qnetd server should be  neutral to all cluster nodes.", which will be generally sensible enough.

Reply:
Hmn, I see your point, but isn't "location" and "should be neutral" a bit, well, too vague? I have the feeling there is something missing. Maybe could we have something like the following?

> A network address which can, but does not have to be part of the cluster. The location of
the QNetd server should be neutral to all cluster nodes.

Reply:
Or: "A network address used by the QNetd server which could be neutrally reached by all cluster nodes"?

Reply:
Much better, but I'm confused by the word "neutrally". What does that mean? Can we omit it without loosing meaning?

Reply:
As kind of an "arbitrator",  the accessibility to the corosync-qnetd server should be fair to all cluster nodes. That's exactly the reason why we talked about the situation, if cluster nodes spread into two different sub-networks, corosync-qnetd server should reside in a different network from any of the networks that the cluster nodes reside in. Otherwise under split-brain, corosync-qnetd server would certainly always "favor" the cluster partition "closer" to it, which is not expected.

Reply:
Ok, thanks for this explanation. I already assumed that you choose that word wisely, but wasn't sure. :-) :+1: 

I keep your sentence. :-)